styles/components/_MenuOptions.scss

```diff
@@ -0,0 +1,29 @@
+.container {
```

Comment:
camelCase in scss components files names [wiki](https://github.com/Rednaxela700/CC-Project-2/wiki#styling)